examples/pylab_examples/quiver_demo.py

```diff
@@ -8,76 +8,89 @@
 The workaround is to manually expand the axes.
 
 '''
-from pylab import *
+import matplotlib.pyplot as plt
+import numpy as np
 from numpy import ma
 
-X, Y = meshgrid(arange(0, 2*pi, .2), arange(0, 2*pi, .2))
-U = cos(X)
-V = sin(Y)
+X, Y = np.meshgrid(np.arange(0, 2 * np.pi, .2), np.arange(0, 2 * np.pi, .2))
+U = np.cos(X)
+V = np.sin(Y)
 
-#1
-figure()
-Q = quiver(U, V)
-qk = quiverkey(Q, 0.5, 0.92, 2, r'$2 \frac{m}{s}$', labelpos='W',
-               fontproperties={'weight': 'bold'})
-l, r, b, t = axis()
+# 1
+plt.figure()
+Q = plt.quiver(U, V)
+qk = plt.quiverkey(Q, 0.5, 0.92, 2, r'$2 \frac{m}{s}$', labelpos='W',
+                   fontproperties={'weight': 'bold'})
+l, r, b, t = plt.axis()
 dx, dy = r - l, t - b
-axis([l - 0.05*dx, r + 0.05*dx, b - 0.05*dy, t + 0.05*dy])
+plt.axis([l - 0.05 * dx, r + 0.05 * dx, b - 0.05 * dy, t + 0.05 * dy])
```

Comment:
Ok, not a bug as what this code is doing is not exactly the same as what `margins()` does. I can get it to be fairly close using `plt.margins(0.05, tight=False)`, but it isn't the same. So I guess we will just leave it as is.

As for the spaces around the asterisks, it probably was clearer before. This is the danger of autopep8! The only time I use it is when I am reworking someone else's code who has very little experience with proper code formatting. The intent is not that the autopep8 version gets into version control, the intent is to make it easier for me to read so that I can rewrite it better. Even then, I don't let it run with all of the options on.